File: android/app/src/main/java/com/reactnativewithoutexpo/BiometricModule.java
package com.reactnativewithoutexpo;

import android.app.Activity;
import android.content.Intent;
import android.net.Uri;

import androidx.annotation.NonNull;

import com.facebook.react.ReactInstanceManager;
import com.facebook.react.ReactNativeHost;
import com.facebook.react.bridge.Callback;
import com.facebook.react.bridge.CatalystInstance;
import com.facebook.react.bridge.Promise;
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;
import com.facebook.react.bridge.ReactMethod;
import com.facebook.react.bridge.WritableNativeArray;

public class BiometricModule extends ReactContextBaseJavaModule {

    BiometricModule(ReactApplicationContext reactContext) {
        super(reactContext);
    }

    @Override
    public String getName() {
        return "BiometricModule";
    }
    
    @ReactMethod
    void navigateToMultiModalActivity() {
        Activity activity = getCurrentActivity();
        if (activity != null) {
            Intent intent = new Intent(activity, com.reactnativewithoutexpo.multibiometric.multimodal.MultiModalActivity.class);
            activity.startActivity(intent);
        }
    }
}

File: android/app/src/main/java/com/reactnativewithoutexpo/BiometricModule.kt
package com.reactnativewithoutexpo

import android.app.Activity
import android.content.Intent
import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.bridge.ReactContextBaseJavaModule
import com.facebook.react.bridge.ReactMethod
//import com.reactnativewithoutexpo.multibiometric.multimodal.MultiModalActivity // Ensure you have the correct import for MultiModalActivity

class BiometricModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {

    override fun getName(): String {
        return "BiometricModule"
    }

//    @ReactMethod
//    fun navigateToMultiModalActivity() {
//        val activity: Activity? = currentActivity
//        if (activity != null) {
//            val intent = Intent(activity, MultiModalActivity::class.java)
//            activity.startActivity(intent)
//        }
//    }
}
